packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -442,43 +524,51 @@ export default function ProjectNav({
         {!isGovernment && !isCloudManager && (
           <>
             <li role="none" className={productTabStyle}>
-              <a
-                data-testid="project-nav-realm"
-                href={projectNav.realm}
-                className={getProductClassName('realm')}
-                aria-disabled={!current}
-                tabIndex={current ? 0 : -1}
-                onClick={onElementClick(ProjectNavRealm)}
-              >
-                {!isMobile && (
-                  <RealmIcon
-                    {...productIconProp.prop}
-                    active={activeProduct === Product.Realm && isLoading}
-                    className={iconStyle}
-                  />
-                )}
-                Realm
-              </a>
+              {useCNRegionsOnly && (
+                <WithCNRegionsOnlyTooltip>
+                  <div className={getProductClassName(Product.Realm)}>
+                    {getRealmIcon()}
+                    Realm
+                  </div>
+                </WithCNRegionsOnlyTooltip>
+              )}
+              {!useCNRegionsOnly && (
```

Comment:
hm, is there a way we can get both branches to be defined in `WithCNRegionsOnlyTooltip` ?

Reply:
It doesn't seem to render properly when I put both branches in `WithCNRegionsOnlyTooltip`. Also, just wondering why we want to render the tooltip if `useCNRegionsOnly == false`?

Reply:
I was thinking the semantics of it would be more like `WithTooltipIfCNRegionsOnly` and handle the branching so we don't need to repeat it for Realm and Charts

Reply:
Ah got it 😄 I'll update the PR to handle this